vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.jsp

```diff
@@ -5,7 +5,7 @@
 <script>
 //# sourceURL=click.js
 var contentCount = ${fn:length(slideContents)};
-
+var imageToReplace = false
```

Comment:
instead of introducing another global variable, just make sure imgID is reset after an image has been uploaded.

Reply:
or the uploadImage data attribute